From 4170240d08b60e84b03fe71c0a711a46b754bb31 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:28:03 -0400
Subject: [alsa] cs553*: Fix dependencies

Message-id: <140972823573-234-patchset-jkysela@redhat.com>
Patchwork-id: 90934
O-Subject: [RHEL 7.1 PATCH 233/442] ALSA: cs553*: Fix dependencies
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

The CS5530, CS5535 and CS5536 chipsets are companions of the Geode
series of processors, which are 32-bit x86 processors. So the
snd-cs5530 and snd-cs5535audio drivers are only needed on this
architecture, except for build testing purpose.

Signed-off-by: Jean Delvare <jdelvare@suse.de>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 5be50ac26635ba64339912943a88899a1f55f8e6)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/Kconfig b/sound/pci/Kconfig
index 9751429..e850e08 100644
--- a/sound/pci/Kconfig
+++ b/sound/pci/Kconfig
@@ -280,7 +280,7 @@ config SND_CS46XX_NEW_DSP
 
 config SND_CS5530
  tristate "CS5530 Audio"
- depends on ISA_DMA_API
+ depends on ISA_DMA_API && (X86_32 || COMPILE_TEST)
  select SND_SB16_DSP
  help
    Say Y here to include support for audio on Cyrix/NatSemi CS5530 chips.
@@ -290,6 +290,7 @@ config SND_CS5530
 
 config SND_CS5535AUDIO
  tristate "CS5535/CS5536 Audio"
+ depends on X86_32 || COMPILE_TEST
  select SND_PCM
  select SND_AC97_CODEC
  help
-- 
1.7.1